api/tests/identity-access-management/integration/domain/usecases/update-user-for-account-recovery.usecase.test.js

```diff
@@ -52,4 +46,154 @@ describe('Integration | Identity Access Management | Domain | UseCase | update-u
       expect(authenticationMethod[0].password).to.equal(authenticatedMethod.password);
     });
   });
+  context('success cases', function () {
```

Comment:
Suggestion : Sauter une ligne.

```suggestion

  context('success cases', function () {
```